packages/provider_devtools_extension/lib/src/provider_screen.dart

```diff
@@ -12,19 +13,22 @@ import 'instance_viewer/instance_providers.dart';
 import 'instance_viewer/instance_viewer.dart';
 import 'provider_list.dart';
 import 'provider_nodes.dart';
+import 'instance_viewer/eval.dart';
 
-final _hasErrorProvider = Provider.autoDispose<bool>((ref) {
-  if (ref.watch(sortedProviderNodesProvider) is AsyncError) return true;
+final _hasErrorProvider = Provider.autoDispose<AsyncError?>((ref) {
```

Comment:
The name of the provider feels odd now.

And we probably shouldn't be returning AsyncError itself, but just the error/stack